### ISSUE TITLE
Chapter 4 issues

### ISSUE THREAD

**Issue by bonnfire** [Amherst-Statistics/IPS9inR#11]
When introducing rbinom() and runif(), should I specify the arguments (i.e. x=, prob=, etc.)? In the code chunk or in the text after the code chunk?

**Comment by nicholasjhorton** [comment]
Please name these arguments.  After usage, you can call x x and size n for rbinom().  You'll also want to add in a simple example of xpbinom().

Nick


> On Jul 17, 2018, at 3:57 PM, Bonnie Lin <notifications@github.com> wrote:
> 
> When introducing rbinom() and runif(), should I specify the arguments (i.e. x=, prob=, etc.)? In the code chunk or in the text after the code chunk?
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

Nicholas Horton 
Professor of Statistics
Department of Mathematics and Statistics, Amherst College
PO Box 5000, AC #2239
Amherst, MA 01002-5000

**Comment by bonnfire** [comment]
Thanks for the response! I will specify the arguments in the function call.

**Comment by nicholasjhorton** [comment]
That sounds like a good plan.

Nick


> On Jul 17, 2018, at 4:25 PM, Bonnie Lin <notifications@github.com> wrote:
> 
> Another question that I had was regarding the Benford's law histogram. Would you like me to use the Benford's base R function to plot the probability histogram?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

Nicholas Horton 
Professor of Statistics
Department of Mathematics and Statistics, Amherst College
PO Box 5000, AC #2239
Amherst, MA 01002-5000

**Comment by bonnfire** [comment]
Goal is to recreate these two plots: 
![figure4 5](https://user-images.githubusercontent.com/22698310/42900035-d697713e-8a95-11e8-8b3a-89cb3aa8a2a6.jpg)
For the first one, I've tried runif and rbinom, specifying 2000 trials, selecting 1 each trial, each with a prob of success = 1/9 or 0.1111.. that doesn't seem to work? 

For the second one, I've written this function 
benlaw <- function(d){
  log10(1 + 1 / d)
}
digits <- 1:9
data <- data.frame(Outcome = benlaw(digits))

However, neither the bar or histogram plots match the one in the textbook

**Comment by nicholasjhorton** [comment]
We may need to defer these questions until my return unless Margaret and Shukry can assist (I presume that you've spoken with them).  

But see:

xpbinom(c(1,10), size=15, prob=0.5)

for a better example

**Comment by bonnfire** [comment]
So I'll just include an image from the WIki page then?

**Comment by nicholasjhorton** [comment]
No. Instead why don't you use the probabilities from the table.  

Nick


> On Jul 18, 2018, at 12:33 PM, Bonnie Lin <notifications@github.com> wrote:
> 
> So I'll just include an image from the WIki page then?
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
> 

Nicholas Horton 
Professor of Statistics
Department of Mathematics and Statistics, Amherst College
PO Box 5000, AC #2239
Amherst, MA 01002-5000

**Comment by nicholasjhorton** [comment]
Any updates on this front?
